website/docs/d/cos_replication.html.markdown

```diff
@@ -0,0 +1,82 @@
+---
+subcategory: "Object Storage"
+layout: "ibm"
+page_title: "IBM : Cloud Object Storage Replication"
+description: |-
+  Get information about an object in an IBM Cloud Object Storage Replication bucket.
+---
+
+# ibm_cos_bucket_replication_rule
+
+Retrieves information of replication configuration on an existing bucket. .  For more information, about configuration options, see [Replicating objects](https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-replication-overview). 
+
+To configure a replication policy on a bucket, you must enable object versioning on both source and destination buckets by using the [Versioning objects](https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-versioning).
+
+## Example usage
+
+```terraform
+data "ibm_resource_group" "cos_group" {
+    name = "cos-resource-group"
+}
+
+data "ibm_resource_instance" "cos_source_instance" {
+    name              = "cos-instance"
+    resource_group_id = data.ibm_resource_group.cos_group.id
+    service           = "cloud-object-storage"
+}
+
+data "ibm_cos_bucket" "source_bucket" {
+    resource_instance_id = data.ibm_resource_instance.cos_source_instance.id
+    bucket_name          = "my-bucket-src"
+    bucket_type          = "region_location"
+    bucket_region        = "us-east"
+}
+
+data "ibm_resource_instance" "cos_destination_instance" {
+    name              = "cos-instance-dest"
+    resource_group_id = data.ibm_resource_group.cos_group.id
+    service           = "cloud-object-storage"
+}
+
+data "ibm_cos_bucket" "destination_bucket" {
+    resource_instance_id = data.ibm_resource_instance.cos_destination_instance.id
+    bucket_name          = "my-bucket-dest"
+    bucket_type          = "region_location"
+    bucket_region        = "us-east"
+}
+
+data "ibm_iam_authorization_policies" "testacc_ds_authorization_policy" {
+}
+
+data "ibm_cos_bucket_replication_rule" "cos_bucket_repl" {
```

Comment:
This is datasource  repliaction_rules is computed can't be given as input

Can you run this complete example as tempalte and check does the example works

Reply:
yes , it gives correct output.